doc/help_modification.md

```diff
@@ -1,6 +1,35 @@
-Modify an existing help
--------------------------
-Each command might have its own help definition. This definition is composed of various _help items_, which might contain their own definitions.
+# Modify an existing help
+
+### Help items
+Before modifying help, it's good to know which help items are available to use.
+There is a list with currently available items:
+```ruby
+Command.extend_help do |h|
+  # Add a simple text to the help output.
+  # content - String - text to be shown
+  # options - Hash   - options (id:, richtext:)
```

Comment:
`richtext:` here means 'bold'?

Reply:
by richtext I'd probably expect something like https://github.com/piotrmurach/tty-markdown ;) Didn't you consider it? Might be next PR candidate ;)